package.json

```diff
@@ -101,6 +101,7 @@
     "standard": "^12.0.1",
     "supertest": "^3.4.2",
     "supertest-session": "^3.3.0",
+    "uglify-js": "^3.4.9",
```

Comment:
What about change `uglify-js` what looks no longer maintained to `terser` who faster?

Reply:
I would be for this, except that in dev mode, `browserify-middleware` uses `uglify-js` internally by default. If we switch this to `terser`, we should also find a way to change it there for consistency. I'll look into it.

Reply:
Made PR `browserify-middleware` (https://github.com/ForbesLindesay/browserify-middleware/pull/126), who changes `uglify-js` to `terser`, if it gets merge, we can move next.

Reply:
That repo isn't seeing a lot of activity, so I'm not certain it'll get merged. As I've been looking and researching, I think I'd ultimately like to remove browserify/browserify-middleware completely and replace it with something else. How do you feel about merging this PR and tackling that in another?